.travis.yml

```diff
@@ -0,0 +1,13 @@
+services:
+    - docker
+
+env:
+    - secure: "jq0r7xvk+XAETGrVRZ3gWgxVlkYTJs+W7WsYHu2VLx6JAweitL4MGgXn45mzgQGw+P4nA/KDZWw0VsvEluvA22kjsq87QtawMm3ZzqIUPe5U7wtdLIvLDSFcOb4SBEh6FfN2uLfLsECv4x4+2LyCZPLHmMuXOpFLd/xatC7wMBhEXt2YPdC2PYZnZ7UQDCNe9OJXzZrRUfLnHuUhBMGtd7hoCWjr84ePeBWg/PC6ec+4p+zajU9DQEzne0e44z7r//qU6O4dfLixke0jgPVBXdVEfSeM2UWL6tCFx+ntEqDR1w/c9qp3lfDEqcye2P6X+2BEqWbL5xv3bySnXvSDm8d3KEoy/XDef7CyfmImcMQGtMi9H1ZPBZ988rJVmJu8jLlrKIsx1VE5+ZekCuT9Ol0atiBWmoyyDd1y8Kumd9UKMicY7h7EeOJtV3b5fvLm1mK3/AEapqjpRrbTa4dKFQyz/7qZp+hJrf2Gf3YlAU8fOZeCDuamrU/3sBbR+vfI5zhErTp74PwgDJwmRAPWsMRb+aQjanD2VjPFND65FjwByvcQ3IOV4NSP3nq4d/Wu95Fzx59JSauTihR3716POaWq5/dzG6IXwdN30le39NJsI8zvl9UFGD0h7VwI2UeKODlLjGrgJ20UhiOJ3GdWmk1/E3B7iuGdJBrGE5kG0s4="
+before_install:
+    - docker build --target test --tag todo-app:test .
```

Comment:
Slightly misleading to do this in a `before_install`. 

The main point of splitting the script into different sections like this is to make it easier to read and understand. `before_install` is run "before the install phase", but you don't have an install phase. I feel like this should be `install` instead, or even just put it in the the `script`

(A functional difference putting it in the "script" is Travis will consider it a build "failure" rather than "error" if the given command returns a non-zero exit code)